website/docs/r/event_orchestration_unrouted.html.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_event_orchestration_unrouted"
+sidebar_current: "docs-pagerduty-resource-event-orchestration-unrouted"
+description: |-
+  Creates and manages an Unrouted Orchestration for a Global Event Orchestration in PagerDuty.
+---
+
+# pagerduty_event_orchestration_unrouted
+
+An Unrouted Orchestration allows users to create a set of Event Rules that will be evaluated against all events that don't match any rules in the Orchestration's Router.
+
+The Unrouted Orchestration evaluates events sent to it against each of its rules, beginning with the rules in the "start" set. When a matching rule is found, it can modify and enhance the event and can route the event to another set of rules within this Unrouted Orchestration for further processing.
+
+## Example of configuring Unrouted Rules for an Orchestration
+
+In this example of an Unrouted Orchestration, the rule matches only if the condition is matched.
+Alerts created for events that do not match the rule will have severity level set to `info` as defined in `catch_all` block.
+`catch_all` for Unrouted Orchestration always suppressess the alert so it is not required to define the `suppress` action.
```

Comment:
I'd argue it's more like "events that are sent to the Unrouted Orchestration will never belong to a service & therefore will **always** be suppressed, `catch_all` or not!" I'm going to remove this sentence for now, but feel free to add it (or a variation) back if you think it's removal makes things unclear. 👍 

```suggestion
```